### DISCUSSION TITLE
2 Questions

### DISCUSSION THREAD

**Discussion by johnstonesnow** [organicmaps/organicmaps#7374] [Q&A]
1. Is it possible to display the current speed limit at all times? I tried to find an option but can't find any. OSMand does it and it forced me to use that but I prefer Organic Maps so would love (and need) that feature

2. Is there any way to review the routes before committing to navigate one? I do a regular journey and based on roadworks there are at least 4 different ways to get from A to B, I can't see a way be able to choose before starting navigation.
thanks

**Comment by biodranik** [comment]
1. Are you using iOS or Android version? Are you talking about a "free drive" mode, not a Navigation mode?
2. Are you talking about #1183 ?

**Comment by johnstonesnow** [comment]
2. Yes - Sorry I struggle to even post on github let alone understand how it all works
1. Android version (Fdroid)
